### PR TITLE
add benchmark flow

### PR DESCRIPTION
**Pull request by JesseAbram**
This PR moves all benchmarks to be run together specifically for statemint/mine. running the bash script from the root will generate all benchmarks.

What still needs to be done is choose the spec of the machine to run and choose the proper steps and repeats, then run them and replace the current weight files.

Partially covers #37

### CODE REVIEW

File: pallet_assets.rs [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::Weight};
+use sp_std::marker::PhantomData;
+
+/// Weight functions for pallet_assets.
+pub struct WeightInfo<T>(PhantomData<T>);
+impl<T: frame_system::Config> pallet_assets::WeightInfo for WeightInfo<T> {
+	fn create() -> Weight {
+		(68_000_000 as Weight)
```
Comment (kianenigma) [inline pallet_assets.rs:40]: these numbers are now from your local machine right?

Comment (JesseAbram) [reply pallet_assets.rs:40]: yup, I put it in the readme and #37 that we need to run it on a proper machine when we are ready

File: scripts/benchmarks.sh [hunk 2]
```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+steps=20
+repeat=10
+statemineOutput=./runtime/statemine/src/weights/
+statemintOutput=./runtime/statemint/src/weights/
+statemineChain=statemine-dev
+statemintChain=statemint-dev
+
+./target/release/statemint benchmark \
```
Comment (kianenigma) [inline scripts/benchmarks.sh:10]: You can surely use bash loops here. an array of pallets and just having the commend once is much better IMO

**Review by kianenigma — APPROVED** [review]

File: pallet_assets.rs [hunk 3]
```diff
@@ -0,0 +1,169 @@
+
+//! Autogenerated weights for pallet_assets
```
Comment (joepetrowski) [inline pallet_assets.rs:2]: Why is this file in the root directory?

Comment (JesseAbram) [reply pallet_assets.rs:2]: oh that was careless, good spot

**Pull request merged**